### PR TITLE
fix airloop index problem in std. 62.1 calculations

### PR DESCRIPTION
**Pull request by EnergyArchmage**
Pull request overview
---------------------
 - Fixes TBD
 - Interface developer noticed problems in the predefined table System Ventilation Requirements for Cooling when the order of input objects was changed.  Found three places where the wrong index was being used. 
 - DESCRIBE PURPOSE OF THIS PULL REQUEST

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [x] Perform a Code Review on GitHub
 - [x] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [x] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [x] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [x] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/SizingManager.cc [hunk 1]
```diff
@@ -1549,7 +1549,7 @@ void ManageSystemVentilationAdjustments(EnergyPlusData &state)
             state.dataSize->VpzClgSumBySys(AirLoopNum) = 0.0;
             // Fill Vps for cooling VRP calculation, use cooling design flow rate as adjusted in ManageSystemSizingAdjustments ( to use
             // conincident sizing result if available for block air flow
-            state.dataSize->VpsClgBySys(AirLoopNum) = FinalSysSizing(SysSizNum).DesCoolVolFlow;
+            state.dataSize->VpsClgBySys(AirLoopNum) = FinalSysSizing(AirLoopNum).DesCoolVolFlow;
```
Comment (rraustad) [inline src/EnergyPlus/SizingManager.cc:1552]: This certainly corrects an index mistake.

Comment (rraustad) [inline src/EnergyPlus/SizingManager.cc:1552]: This certainly corrects an index mistake.

**Review by rraustad — APPROVED** [review]
These changes look correct. Diffs are not related and err diffs are from recent AFN changes.

**Comment by rraustad** [comment]
@Myoldmopar This is a straight-forward fix, are you OK with a merge here or should develop be pulled once again to eliminate diffs?

**Comment by EnergyArchmage** [comment]
Diff in TranspiredCollecter is expected. Could use that file for defect issue.

**Comment by rraustad** [comment]
This looks like just a reporting issue, at least for TranspiredCollectors. The air loop flow rate did not change. The reported Vps did change as did the percent OA. 

    Component Sizing Information, AirLoopHVAC, DIRECT HEATING AIR LOOP ZN11, Adjusted Heating Design Air Flow Rate [m3/s], 3.00000
    Component Sizing Information, AirLoopHVAC, DIRECT HEATING AIR LOOP ZN11, Adjusted Cooling Design Air Flow Rate [m3/s], 3.00000

And the air flow rate at the zone inlet node and the zone temperature for the zone served by Direct Heating Air Loop ZN11 has also not changed. the air flow rate = 3 m3/s and the results show that flow rate before and after this change are the same. This zone's HVAC was hard sized to 3 m3/s.

    Fan:ConstantVolume,
      Supply Fan ZN11,         !- Name
      ALWAYS_ON,               !- Availability Schedule Name
      0.7,                     !- Fan Total Efficiency
      600,                     !- Pressure Rise {Pa}
      3.0,                     !- Maximum Flow Rate {m3/s}


image

This is for System Ventilation Requirements for Cooling and the code change was for cooling as state.dataSize->VpsClgBySys(AirLoopNum) = FinalSysSizing(AirLoopNum).DesCoolVolFlow; The data in System Ventilation Requirements for Heating showed the correct data and now this cooling table matches the heating table.



The first Sizing:System object is Direct DX air Loop ZN1 which is the data reported for the last 4 air systems. I'm sure there is a reason why this data was not reported for all air systems in this table.

    Sizing:System,
      Direct DX air Loop ZN1,  !- AirLoop Name

**Comment by rraustad** [comment]
I was wondering why the first air loop data was used for the "Direct Heating" air loops while the others seemed to use valid data. These last 4 air loops do not have a Sizing:System object, so the first is used. This all makes sense now.

        if (SysSizNum == 0) SysSizNum = 1; // use first when none applicable

    Sizing:System,
      Direct DX air Loop ZN1,  !- AirLoop Name

    Sizing:System,
      Direct DX air Loop ZN2,  !- AirLoop Name

    Sizing:System,
      Direct DX air Loop ZN3,  !- AirLoop Name

    Sizing:System,
      Direct DX air Loop ZN4,  !- AirLoop Name

    Sizing:System,
      Direct DX air Loop ZN5,  !- AirLoop Name

**Comment by rraustad** [comment]
Pulled develop and unit tests run to completion locally. Pushing this branch to see the change in diffs. This is ready to go.

**Comment by mitchute** [comment]
I pulled in develop again and ran unit tests, which all completed successfully. Diffs are expected, and this looks like a no-brainer fix. Merging. Thanks @EnergyArchmage and @rraustad.

**Pull request merged**